source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
 <h4>UX Changes</h4>
-* <%= link_to_commit '67f1605fca894ea9855d16a283a913335200f3b2', 'Find Value Stream Map (VSM) more easily', 3705%> (<%= link_to_issue 3858%>)
+* <%= link_to_commit '0e926106003a040ad4c62bc7810df0b851a35864', 'Plugins SPA is enabled by default.', 4030%>.
 
-    <%= link_to_commit '9f6909e2f64b07d2dce5cecd4ea5b92b8e19d6b1', 'Renames pipeline \'Label\' to \'Instance\' as part of an ongoing UX refresh.', 3944 %> (<%= link_to_issue 3949%>)
-
-    Pipeline instances have an explict link to VSM across the application for better discoverablity. Moreover, the term pipeline `Label` has been changed to `Instance` as part of an ongoing UX refresh.
-
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-dashboard.png', alt: 'Explicit vsm link in dashboard' %>
+<figure class='shrink-figure-75'>
+      <%= image_tag 'releases/17.12.0/Plugins-SPA.png', alt: 'Plugins SPA' %>
   </figure>
+* <%= link_to_commit '979fa507d3527927919a730bf421e92c2090a972', 'Addition of pipeline count information to about page', 3960%>
```

Comment:
That's not a UX change. Move it as the last item in the other improvements section.